apps/elf-council-frontend/src/ui/gsc/GSCOverviewSection.tsx

```diff
@@ -149,11 +149,22 @@ export function GSCOverviewSection(): ReactElement {
                       <p>{t`Any delegate who has accumulated more delegated voting
                       power than the current GSC Eligibility Threshold (${formattedThreshold}
                       ELFI) is eligible to join the GSC. `}</p>
-                      <p>{t`To help reach this
+                      <p>
+                        {t`To help reach this
```

Comment:
we shouldn't break up a sentence like this using `t`.  Instead, we should use `jt` with the link as a variable since sentence structure can change language to language.